offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/update/UpdateAffiliationView.groovy

```diff
@@ -0,0 +1,130 @@
+package life.qbic.portal.offermanager.components.affiliation.update
+
+import com.vaadin.icons.VaadinIcons
+import com.vaadin.ui.*
+import com.vaadin.ui.themes.ValoTheme
+import groovy.util.logging.Log4j2
+import life.qbic.business.Constants
+import life.qbic.business.persons.affiliation.Affiliation
+import life.qbic.portal.offermanager.components.*
+import life.qbic.portal.offermanager.components.affiliation.AffiliationFormView
+import life.qbic.portal.offermanager.dataresources.ResourcesService
+
```

Comment:
Same here adding Javadocs would be helpful to understand the purpose of this class